app/main.py

```diff
@@ -1,6 +1,21 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+key = os.getenv("API_KEY")
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(
+        "http://api.weatherapi.com/v1/current.json",
+        params={"q": "Paris", "key": key}
```

Comment:
Code Style issue: The URL and the location 'Paris' are also constants and should be defined as such at the beginning of the script. For example: 

URL = 'http://api.weatherapi.com/v1/current.json'
LOCATION = 'Paris'

And then in the requests.get method you can use these constants: 

requests.get(URL, params={'q': LOCATION, 'key': KEY})